documentation/configuration/basic.md

```diff
@@ -5,7 +5,7 @@ description: 'Basic configuration'
 ---
 
 :::info
-_This page assume that you have already completed the [download](/docs/setup/prerequisites)._
```

Comment:
Can you point me to the rules that make this a typo?
I can find examples of both being used like this as valid.

Reply:
Perfect 👍